packages/origin-ui-core/src/components/orders/MyOrders.tsx

```diff
@@ -11,7 +11,9 @@ import { Box, useTheme } from '@material-ui/core';
 
 export const MyOrders = () => {
     const user = useSelector(getUserOffchain);
-    const orders: Order[] = useSelector(getOrders).filter((o) => o.userId === user.id.toString());
+    const orders: Order[] = useSelector(getOrders).filter(
+        (o) => o.userId === user.organization.id.toString()
```

Comment:
Are you sure we need this .filter ? When requesting using GET /orders you will receive **your** orders anyway.

Reply:
You right. I'll remove it